src/lib/styles/app.css

```
@@ -1,6 +1,14 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import 'tailwindcss';
+
+@theme {
+  --font-sans: 'LINESeedJPStd', ui-sans-serif, system-ui, sans-serif;
+  --grid-template-columns-layout: 16rem 1fr 16rem;
+}
+
+@plugin 'daisyui' {
+  /* biome-ignore lint/correctness/noUnknownProperty -- DaisyUI plugin config */
```

Biome (2.1.2)

[error] Failed to parse category "--". Example of suppression: // biome-ignore lint: reason

Biome can't recognize the category, usually it's because it doesn't match the expected ones.
Example of suppression: // biome-ignore lint: reason

(suppressions/parse)